test/3_branch.js

```diff
@@ -194,7 +194,7 @@ describe('Branch', function() {
 		var expectedResponse = {
 			identity_id: '12345',
 			link: 'url',
-			referring_data: { },
+			referring_data: '{ }',
```

Comment:
Actually, no. Because the setIdentity method didn't do anything with the data parameter previously. So it didn't matter what type it was. Not it is required to be a string, because I'm parsing it.